packages/training/deploy/.env.example

```
@@ -0,0 +1,113 @@
+# Image tag for builds
+TAG=latest
+
+# =============================================================================
+# PLATFORM: RUNPOD
+# =============================================================================
+
+# RunPod API key for programmatic pod management
+# Get from: https://runpod.io/console/user/settings
+RUNPOD_API_KEY=
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)